lib/std/mem.zig

```diff
@@ -709,6 +709,14 @@ pub fn zeroInit(comptime T: type, init: anytype) T {
                 .Struct => |init_info| {
                     var value = std.mem.zeroes(T);
 
+                    // typeInfo won't tell us if this is a tuple
```

Comment:
I'm fine with leaving this for now, but I'm thinking we add this to type info in the language spec, yeah?

Reply:
`is_tuple` was accepted as part of #4335, just hasn't been implemented yet.

Reply:
I may not have a great memory, but at least I am consistent :sweat_smile: